### ISSUE TITLE
firebase database:remove throws TypeError [ERR_INVALID_ARG_TYPE]: The "url" argument must be of type string. Received type undefined

### ISSUE THREAD

**Issue by JustAnotherGitHubUserYouDontKnow** [firebase/firebase-tools#2667]
### [REQUIRED] Environment info

**firebase-tools:** 8.11.2

**Platform:** Linux Mint

### [REQUIRED] Test case

Via the CLI (logged in, of course):

```firebase database:remove /foobar```

where /foobar exists in the (realtime) database, and I get:

```Error: An unexpected error has occurred.```

and firebase-debug.log shows:

```
[debug] [2020-10-02T22:18:58.293Z] TypeError [ERR_INVALID_ARG_TYPE]: The "url" argument must be of type string. Received type undefined
    at Url.parse (url.js:154:11)
    at urlParse (url.js:148:13)
    at Url.resolve (url.js:663:29)
    at Object.urlResolve [as resolve] (url.js:659:40)
    at Object.getDatabaseUrl (/usr/local/lib/node_modules/firebase-tools/lib/utils.js:62:26)
    at Command.actionFn (/usr/local/lib/node_modules/firebase-tools/lib/commands/database-remove.js:33:32)
    at Command.<anonymous> (/usr/local/lib/node_modules/firebase-tools/lib/command.js:166:25)
    at Generator.next (<anonymous>)
    at fulfilled (/usr/local/lib/node_modules/firebase-tools/lib/command.js:5:58)
    at process._tickCallback (internal/process/next_tick.js:68:7)
[error] 
[error] Error: An unexpected error has occurred.
```

### [REQUIRED] Steps to reproduce

Manually add a key value pair e.g. Name: ```foobar``` Value: ```baz``` to the root of your realtime database e.g. via the interface at https://console.firebase.google.com/, then via the CLI: ```firebase login``` and ```firebase database:remove /foobar```

### [REQUIRED] Expected behavior

The key at /foobar (and it's value) should be removed from the database

### [REQUIRED] Actual behavior

The CLI quits with the error ```Error: An unexpected error has occurred.``` and a similar output above in the log

**Comment by JustAnotherGitHubUserYouDontKnow** [comment]
(Also experienced by another user on stackoverflow: https://stackoverflow.com/questions/64135814/cant-remove-values-in-firebase-realtime-database?newreg=24c19ece7f1d458c8079226b5decfdd8 - but I have no reps to comment there)

**Comment by vladkras** [comment]
same error

**Comment by JustAnotherGitHubUserYouDontKnow** [comment]
Resolved with [Changelog for database:remove command fix #2658](https://github.com/firebase/firebase-tools/pull/2658) merged into master and released in [version 8.12.0](https://github.com/firebase/firebase-tools/releases/tag/v8.12.0)

Works great 👍
